WebView/sampleData/weather.json

```diff
@@ -0,0 +1,29 @@
+{
+  "newYork":{
+    "currentTemp":"75°",
+    "highTemp":"83°",
+    "lowTemp":"64°",
+    "chancePrecip":"90%",
+    "humidity":"23%",
+    "description":"Rainy",
+    "icon":"https://gcoleman799.github.io/res/drawable/rain.png"
```

Comment:
Can we move the icon URL out of the json? I feel like the app should have logic to choose the icon based on the description.